### PR TITLE
Aleksandar-MessageSendingService

### PR DESCRIPTION
**Pull request by Aleksandar1105**

### CODE REVIEW

File: Shipfinity.Domain/Models/Message.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.Domain.Models
+{
+    public class Message : BaseEntity
+    {
+        public int Id { get; set; }
+        public string FirstName { get; set; }
```
Comment (KristijanJankuloski) [inline Shipfinity.Domain/Models/Message.cs:12]: Add restrictions

File: Shipfinity.DataAccess/Repositories/Interfaces/IMessageRepository.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+﻿using Shipfinity.Domain.Models;
+
+namespace Shipfinity.DataAccess.Repositories.Interfaces
+{
+    public interface IMessageRepository : IRepository<Message>
+    {
+        Task<Message> CreateMessageAsync(Message message);
```
Comment (KristijanJankuloski) [inline Shipfinity.DataAccess/Repositories/Interfaces/IMessageRepository.cs:7]: Move this to Insert Async

File: Shipfinity.Domain/Models/Message.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.Domain.Models
+{
+    public class Message : BaseEntity
+    {
+        public int Id { get; set; }
```
Comment (KristijanJankuloski) [inline Shipfinity.Domain/Models/Message.cs:11]: Remove id

File: Shipfinity.Api/Controllers/MessageController.cs [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+        public MessageController(IMessageService messageService)
+        {
+            _messageService = messageService;
+        }
+
+        [HttpPost]
+        public async Task<IActionResult> SendMessage(MessageDto messageDto)
+        {
+            if(!ModelState.IsValid)
```
Comment (KristijanJankuloski) [inline Shipfinity.Api/Controllers/MessageController.cs:23]: Wrap in try catch

**Review by KristijanJankuloski — CHANGES_REQUESTED** [review]

**Comment by KristijanJankuloski** [comment]
Revert changes to csproj file to resolve merge conflict

**Review by cheppymk — APPROVED** [review]

**Review by KristijanJankuloski — APPROVED** [review]

**Pull request merged**